deny.toml

```
@@ -91,7 +91,7 @@ skip = [
 
 skip-tree = [
     # reqwest is using several older dependencies
-    { name = "reqwest", version = "=0.11" },
+    { name = "reqwest", version = "=0.12" },
```

```suggestion
    { name = "reqwest", version = "=0.11" },
```
I think we want to keep this 0.11 since the older `reqwest` is the problematic dependency (and hopefully one we can get rid of soon).

---

You're right - looks like we still rely on reqwest 0.11 via `noxious-server`. I looked more closely and it appears `aws-config` is problematic now, as it brings in an older version of `rustls` and friends. I'm able to resolve with

```suggestion
{ name = "reqwest", version = "=0.11" },
# aws-config is using several older dependencies
{ name = "aws-config", version = "=1.5" },
```

WDYT?